src/ccall/ccall.c

```diff
@@ -1369,6 +1369,26 @@ int  ccall_request_video_streams(struct icall *icall,
 		goto out;
 	}
 
+    if (list_count(clientl) == 1 ) {
```

Comment:
@z-dule I think you are right it has to be bigger then 1.. is the current local client as well in this List?